examples/material_gallery/lib/demo/menu_demo.dart

```diff
@@ -94,7 +94,7 @@ class MenuDemoState extends State<MenuDemo> {
                   child: new Text('Context menu item one')
                 ),
                 new PopupMenuItem(
-                  disabled: true,
+                  isDisabled: true,
```

Comment:
I've changed isDense to dense. Sadly, I've just noticed that Input also supports dense layout. It's property is called isDense. Sigh.